subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1425,6 +1516,128 @@ static void le_remove_cig(struct net_buf *buf, struct net_buf **evt)
 
 #endif /* CONFIG_BT_CENTRAL */
 
+#if defined(CONFIG_BT_CTLR_CENTRAL_ISO) || \
+	defined(CONFIG_BT_CTLR_PERIPHERAL_ISO)
+static void le_setup_iso_path(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_cp_le_setup_iso_path *cmd = (void *)buf->data;
+	struct bt_hci_rp_le_setup_iso_path *rp;
+	uint8_t status;
+	uint16_t handle;
+	uint16_t company_id;
+	uint16_t vendor_id;
+	uint32_t controller_delay;
+	uint8_t *codec_config;
+
+	handle     = sys_le16_to_cpu(cmd->handle);
+	company_id = sys_le16_to_cpu(cmd->company_id);
+	vendor_id  = sys_le16_to_cpu(cmd->vendor_id);
+	controller_delay = sys_get_le24(cmd->controller_delay);
+	codec_config = &cmd->codec_config[0];
+
+	status = ll_setup_iso_path(handle, cmd->path_dir, cmd->path_id,
+				   cmd->coding_format, company_id, vendor_id,
+				   controller_delay, cmd->codec_config_len,
+				   codec_config);
+
+	rp = hci_cmd_complete(evt, sizeof(*rp));
+	rp->status = status;
+	rp->handle = cmd->handle;
+}
+
+static void le_remove_iso_path(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_cp_le_remove_iso_path *cmd = (void *)buf->data;
+	struct bt_hci_rp_le_remove_iso_path *rp;
+	uint8_t status;
+	uint16_t handle;
+
+	handle = sys_le16_to_cpu(cmd->handle);
+
+	status = ll_remove_iso_path(handle, cmd->path_dir);
+
+	rp = hci_cmd_complete(evt, sizeof(*rp));
+	rp->status = status;
+	rp->handle = cmd->handle;
+}
+
+static void le_iso_receive_test(struct net_buf *buf, struct net_buf **evt)
```

Comment:
Although if adding the broadcaster/receiver, we might want to have additional checks for the `receive` and `transmit` for the receiver and broadcaster respectively

Reply:
Will leave this for a subsequent PR